### PR TITLE
updated docker_componse.yml

### PR DESCRIPTION
**Pull request by LoekL**
Aligned with `master` following https://github.com/spatialos/online-services/pull/113

### CODE REVIEW

File: services/csharp/IntegrationTest/docker_compose.yml [hunk 1]
```diff
@@ -77,16 +70,9 @@ services:
       - "D6DE8"
       - --spatial_project
       - "${SPATIAL_PROJECT}"
-networks:
-      - compose-integrationtest-network
+
```
Comment (JoelOtter) [inline services/csharp/IntegrationTest/docker_compose.yml]: are these newlines supposed to be here?

Comment (LoekL) [reply services/csharp/IntegrationTest/docker_compose.yml]: I'd say yes for the one on line 56, as it's also there in `master`: https://github.com/spatialos/online-services/blob/master/services/csharp/IntegrationTest/docker_compose.yml#L56

The one on 73 was a setting of my text editor..: https://github.com/atom/atom/issues/4741 - removed it!

File: services/csharp/IntegrationTest/docker_compose.yml [hunk 2]
```diff
@@ -77,16 +70,9 @@ services:
...
     environment:
       SPATIAL_REFRESH_TOKEN: "${SPATIAL_REFRESH_TOKEN}"
-      GATEWAY_SERVICE_TARGET: "gateway-internal:4040"
-    networks:
-      - compose-integrationtest-network
-
-networks:
-  compose-integrationtest-network:
-    name: compose-integrationtest-network
+      GATEWAY_SERVICE_TARGET: "gateway-internal:4040"
```
Comment (LoekL) [inline services/csharp/IntegrationTest/docker_compose.yml]: @JoelOtter - now GitHub is actually complaining: "No newline at end of file"

Comment (JoelOtter) [reply services/csharp/IntegrationTest/docker_compose.yml]: oh we should have newlines at the end. I meant the ones you added into the middle of the file

Comment (LoekL) [reply services/csharp/IntegrationTest/docker_compose.yml]: fixed!

**Pull request merged**